source/includes/connect/stable-api.php

```diff
@@ -0,0 +1,23 @@
+// start-specify-v1
+<?php
+
+$uri = "mongodb://<hostname>:<port>";
+
+$driverOptions = [
+    'serverApi' => new MongoDB\Driver\ServerApi('1')
+];
+
+$client = new MongoDB\Client($uri, [], $driverOptions);
+// end-specify-v1
+
+// start-stable-api-options
+<?php
+
+$uri = "mongodb://<hostname>:<port>";
+
+$driverOptions = [
+    'serverApi' => new MongoDB\Driver\ServerApi('1', true, true)
```

Comment:
@alcaeus: I'm not sure what version of PHPLIB is being targeted here, but if we're bumping to PHP 8.1+ with version 1.20 I think this would be a good opportunity to use [named arguments](https://www.php.net/manual/en/functions.arguments.php#functions.named-arguments) (available in PHP 8.0+) for the the booleans. Consider:

```
'serverApi' => new MongoDB\Driver\ServerApi('1', strict: true, deprecationErrors: true),
```

Ignore this suggestion if we need to remain compatible with earlier versions (back to PHP 7.4).

Also, my previous comment about a trailing comma after elements in multi-line arrays also applies here.

Reply:
I think we can assume newer PHP versions, so using named arguments sounds like a great idea, especially for those boolean flags!

Reply:
love it. helps a ton with readability.